EIPS/eip-747.md

```diff
@@ -105,61 +117,15 @@ ethereum.request({
+Upon calling this request, the user must be prompted with the opportunity to add this token to their wallet:
 
 ![add-token-prompt 1](../assets/eip-747/add-token-prompt.gif)
 
-For `Base64` images, the user just have to add it as `image` parameter.
-
-```javascript
-const base64image = 'data:image/png;base64, ... ';
-ethereum.request({
-  method: 'wallet_watchAsset',
-  params: {
-    type: 'ERC20',
-    options: {
-      address,
-      symbol,
-      decimals,
-      image: base64image,
-    },
-  },
-});
-```
-
-Upon calling this request, the user should be prompted with the opportunity to add this token to their wallet:
-
-![add-token-prompt 2](../assets/eip-747/add-token-prompt2.gif)
-
-If the user adds this token, it should appear somewhere in their wallet's UI, with its balance, etc.
-
-As a result of the addition or not of the asset a `Promise` should be returned, indicating if the user added the asset or an error if some parameter is not valid.
-
-If the wallet does not support the specified asset type, the error should indicate this with the message:
-
-```javascript
-`Asset of type '${type}' not supported`;
-```
-
-## Rationale
-
-Displaying a user's assets is a basic feature that every modern dapp user expects. However, keeping this list, and polling for it from the network can be costly, especially on bandwidth constrained devices.
-
-Most wallets today either manage their own assets list, which they store client side, or they query a centralized API for balances, which reduces decentralization, letting that API's owner easily correlate account holders with their IP addresses.
-
-Maintaining one of these assets lists becomes a political act, and maintainers can be subject to regular harassment and pressure to list otherwise unknown assets.
-
-Furthermore, automatically listing assets makes assets into a sort of spam mail: Users suddenly seeing new assets that they don't care about in their wallet can be used to bombard them with information that they didn't opt into.
-
-This phenomenon is exacerbated by the trend towards airdropped tokens, which has been a cause of network congestion, because spamming people with new tokens has so far been rewarded with user attention.
-
-While some people might suggest we begin a TCR of trusted tokens to watch, this would not solve the client-side bandwidth issues, nor the airdropped token spam issues. What we really want is a small list of tokens the user cares about.
-
-Most of the time a user is adding a asset, they learned about it on a website. At that moment, there is a natural alignment of interests, where the website wants the user to track their asset, and the user wants to track it. This is a natural point to introduce an API to easily allow these parties to collaborate, without involving the politics of the wallet's developers.
+## Security Considerations
 
-## Implementation
+### SSRF
```

Comment:
See https://www.youtube.com/watch?v=RCJdPiogUIk for an explanation as to why this is an issue